website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,134 @@
+
+### Applied filters
+
+The **applied filters** displays which filter values have been applied (if any) and provides a user with a method to clear or reset all filters at once. In most circumstances the filters that have been applied should be represented using a [Tag](/components/tag) component which allows for the individual dismissal of a filter value.
+
+![Applied filters](/assets/patterns/filter-patterns/applied-filters.png)
+
+#### Positioning
+
+Applied filters should be positioned as close as possible relative to the data set to help the user understand what has been filtered upon. In practice this is most often directly above the data set.
+
+**Ex 2.0: Between the filter bar and the data set**
+
+![Applied filters with filter bar positioning](/assets/patterns/filter-patterns/applied-filters-positioning-filter-bar.png)
+
+**Ex 2.1: Above the data set with filters positioned in a sidebar**
+
+![Applied filters with filter sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar.png)
+
+If the filters are positioned in a page-level sidebar the layout can be simplified by excluding the applied filters and including an action to clear all of the filters within the sidebar itself.
+
+With this method, the filter values being applied to the dataset are already displayed and implied in the sidebar.
+
+**Ex 2.2: Excludes the applied filters when positioned in a sidebar**
+
+![Applied filters with clear all in the sidebar](/assets/patterns/filter-patterns/applied-filters-positioning-sidebar-clear-all.png)
+
+### Pagination
+
+Use pagination to break down the filtered data set into pages. For more details refer to the [Pagination](/components/pagination) guidelines.
+
+![Pagination example](/assets/patterns/filter-patterns/pagination-layout.png)
+
+## Overflow
+
+Depending on the complexity of the data set or filterable parameters, it may be necessary to account for overflow of elements within the filter bar or in the applied tags.
+
+### Filter bar overflow
+
+If the number of filterable parameters exceeds the available space, consider prioritizing the most commonly used filters in the filter bar and moving less important or more complex filters to a [Flyout](/components/flyout) that can be triggered via an action in the filter bar.
+
+**Ex 3.0: Hiding additional filters behind an action; in this scenario a secondary button**
+
+![More filters trigger](/assets/patterns/filter-patterns/overflow-filter-bar.png)
+
+**Ex 3.1: Triggering a Flyout with additional filter parameters**
+
+![Flyout triggered by more filters](/assets/patterns/filter-patterns/overflow-filter-bar-flyout.png)
+
+!!! Dont
+
+Don't stack multiple filter bars on top of a data set. This can result in unnecessary complexity in the UI.
+
+![Multiple filter bar rows](/assets/patterns/filter-patterns/overflow-filter-bar-multiple-rows.png)
+
+!!!
```

Comment:
I think this would be more impactful if it weren't completely out of the realm of possibility. For example, it might be good to showcase an example we've seen recently of a more complex filtering variation, just done incorrectly. This "don't" example is really supposed to showcase stacking filters, no multiple search fields. It just feels like it's doing too much.